frontend/src/metabase/parameters/components/ParameterValueWidget.tsx

```diff
@@ -205,7 +205,7 @@ export const ParameterValueWidget = ({
       {...popoverProps}
     >
       <Popover.Target>
-        <Box onClick={toggle}>
+        <Box onClick={toggle} className={CS.cursorPointer}>
```

Comment:
`Box` supports a [style prop](https://mantine.dev/styles/style/). I don't know what's our currently established best practice, but maybe that's a preferred solution here?

Reply:
I'm under the impression that we prefer to use CSS module classes over style props. I'll loop @deniskaber in since he's the styles-guy.